hbase-http/src/main/java/org/apache/hadoop/hbase/http/ProfileServlet.java

```diff
@@ -355,10 +358,10 @@ private Output getOutput(final HttpServletRequest req) {
       try {
         return Output.valueOf(outputArg.trim().toUpperCase());
       } catch (IllegalArgumentException e) {
-        return Output.SVG;
+        return Output.HTML;
```

Comment:
officially, async profiler 2 supports output format "flamegraph". But looks like "html" does the same.